### PR TITLE
feat: add shutdown docs

### PR DESCRIPTION
**Pull request by Hugome**
- [X] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [X] Commands include options for Linux, MacOS, and Windows within codetabs
- [X] New file and folder names are globally unique
- [X] Page references use shortcodes instead of markdown or URL links
- [X] Images use HTML style and have alternative text
- [X] Places where multiple code/command options are given have codetabs

## Description

Add documentation for dapr/dapr#2828 (New Shutdown API for job)

### CODE REVIEW

**Comment by orizohar** [comment]
@Hugome - When you write "jobs" are you referring to [Kubernetes jobs](https://kubernetes.io/docs/concepts/workloads/controllers/job/)? if this is a Kubernetes only feature it should go under `operations/hosting/kubernetes/`. I would also create more clarity in the title and linkTitle. For someone unfamiliar this can be confusing. 

Suggestion:

...
title: "Running Dapr with a Kubernetes Job"
linkTitle: "Kubernetes  Jobs"
...


Also, it would be good to link (as above) to the definition in the k8s docs in the article to set the context for people landing on this and not sure what "jobs" refers to.

File: daprdocs/content/en/developing-applications/integrations/job.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+title: "Dapr's job integration"
+linkTitle: "job"
+weight: 1000
+description: "Use Dapr API in a job context"
+type: docs
+---
+
+# Job
+
+The Darp sidecar is designed to be a long running process, in the context of a job this behaviour can block your job completion.
```
Comment (orizohar) [inline daprdocs/content/en/developing-applications/integrations/job.md]: "Darp" --> "Dapr"

**Review by orizohar — CHANGES_REQUESTED** [review]
Please see my comment on the location of this file and setting the context for readers

File: daprdocs/content/en/developing-applications/integrations/job.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        dapr.io/app-id: "with-shutdown"
+    spec:
+      containers:
+      - name: job
+        image: busybox
+        command: ["/bin/sh", "-c", "sleep 20 && wget localhost:3500/v1.0/shutdown"]
+      restartPolicy: Never
+```
+
+You can also call the `Shutdown` from any of the Darp SDK
```
Comment (orizohar) [inline daprdocs/content/en/developing-applications/integrations/job.md]: "Darp" --> "Dapr"

**Review by orizohar — CHANGES_REQUESTED** [review]

**Comment by Hugome** [comment]
@orizohar Changes done, I was not sure where to create this page and if I a needed to be really specific about K8S, you have answered :)
And links added to the K8S Job docs

**Review by orizohar — APPROVED** [review]
Thanks for the quick response to feedback. And thanks for the docs contribution!

**Comment by orizohar** [comment]
@yaron2 - can you take a look at this as well?

**Review by orizohar — APPROVED** [review]

**Review by yaron2 — APPROVED** [review]

**Pull request merged**